File: biblioteca/Libro.cs
class Libro
{
    public string Titulo { get; set; }
    public string Autor { get; set; }
    public string Año { get; set; }
    public int Id { get; set; }
    public bool EnPrestamo { get; set; }

    public Libro(int id,string titulo,string autor,string año  )
    {
        Id = id;
        Titulo = titulo;
        Autor = autor;
        Año = año;
        EnPrestamo = false;
    }
}

File: bibliotecapython/libro.py
from prestable import Prestable

class Libro(Prestable):
    def __init__(self, titulo, autor):
        self.__titulo = titulo
        self.__autor = autor
        self.__disponible = True

    @property
    def titulo(self):
        return self.__titulo

    @property
    def autor(self):
        return self.__autor

    @property
    def disponible(self):
        return self.__disponible

    def prestar(self):
        if self.__disponible:
            self.__disponible = False
            return True
        else:
            return False

    def devolver(self):
        self.__disponible = True